testing/render-question-with-cypress.tsx

```diff
@@ -31,12 +31,20 @@ const renderQuestion = (
         [assetKey: string]: boolean;
     } = {};
     const setAssetStatus = (assetKey: string, loaded: boolean) => {
+        console.log("setAssetStatus", assetKey, loaded);
```

Comment:
Should we remove these `console.log` calls before landing? (this one and line 39 below)

Seems like the changes to this file can just be reverted.

Reply:
Good catch. Thanks!